src/riak_kv_ts_svc.erl

```diff
@@ -564,6 +566,11 @@ make_undefined_field_in_insert_resp(BadFields) ->
       ?E_BAD_QUERY,
       flat_format("undefined fields: ~s", [string:join(BadFields, ", ")])).
 
+make_identifier_unexpected_resp(Identifier) ->
```

Comment:
We do not test this bad response and the reason I did not originally provide this function to make a response for the case that the error is `identifier_unexpected` is that `xlate_insert_to_putdata` does not provide a means to report multiple errors, which excludes the ability to report on multiple error types.